pythreejs/pythreejs.py

```diff
@@ -191,374 +167,485 @@ def vector_divide_scalar(self, scalar, x):
+                   'MultiplyOperation').tag(sync=True)
+
+
 class DepthMaterial(Material):
-    _view_name = Unicode('DepthMaterialView', sync=True)
-    wireframe = Bool(False, sync=True)
-    wireframeLinewidth = CFloat(1.0, sync=True)
+    _view_name = Unicode('DepthMaterialView').tag(sync=True)
+    _model_name = Unicode('DepthMaterialModel').tag(sync=True)
+
+    wireframe = Bool().tag(sync=True)
+    wireframeLinewidth = CFloat(1.0).tag(sync=True)
+
 
 class _LineMaterial(Material):
     """Abstract base class for line materials"""
-    _view_name = None
-    
+
+
 class LineBasicMaterial(_LineMaterial):
-    _view_name = Unicode('LineBasicMaterialView', sync=True)
-    color = Color('white', sync=True)
-    linewidth = CFloat(1.0, sync=True)
-    linecap = Unicode('round', sync=True)
-    linejoin = Unicode('round', sync=True)
-    fog = Bool(False, sync=True) 
-    vertexColors = Enum(['NoColors', 'FaceColors', 'VertexColors'], 'NoColors', sync=True)
+    _view_name = Unicode('LineBasicMaterialView').tag(sync=True)
+    _model_name = Unicode('LineBasicMaterialModel').tag(sync=True)
+
+    color = Color('white').tag(sync=True)
+    linewidth = CFloat(1.0).tag(sync=True)
+    linecap = Unicode('round').tag(sync=True)
+    linejoin = Unicode('round').tag(sync=True)
+    fog = Bool().tag(sync=True)
+    vertexColors = Enum(['NoColors', 'FaceColors', 'VertexColors'], 'NoColors').tag(sync=True)
+
 
 class LineDashedMaterial(_LineMaterial):
-    _view_name = Unicode('LineDashedMaterialView', sync=True)
-    color = Color('white', sync=True)
-    linewidth = CFloat(1.0, sync=True)
-    scale = CFloat(1.0, sync=True)
-    dashSize = CFloat(3.0, sync=True)
-    gapSize = CFloat(1.0, sync=True)
-    vertexColors = Enum(['NoColors', 'FaceColors', 'VertexColors'], 'NoColors', sync=True)
-    fog = Bool(False, sync=True)
+    _view_name = Unicode('LineDashedMaterialView').tag(sync=True)
+    _model_name = Unicode('LineDashedMaterialModel').tag(sync=True)
+
+    color = Color('white').tag(sync=True)
+    linewidth = CFloat(1.0).tag(sync=True)
+    scale = CFloat(1.0).tag(sync=True)
+    dashSize = CFloat(3.0).tag(sync=True)
+    gapSize = CFloat(1.0).tag(sync=True)
+    vertexColors = Enum(['NoColors', 'FaceColors', 'VertexColors'], 'NoColors').tag(sync=True)
+    fog = Bool().tag(sync=True)
+
 
 class NormalMaterial(Material):
-    _view_name = Unicode('NormalMaterialView', sync=True)
-    morphTargets = Bool(False, sync=True)
-    shading = Enum(['SmoothShading', 'FlatShading', 'NoShading'], 'SmoothShading', sync=True)
-    wireframe = Bool(False, sync=True)
-    wireframeLinewidth = CFloat(1.0, sync=True)
+    _view_name = Unicode('NormalMaterialView').tag(sync=True)
+    _model_name = Unicode('NormalMaterialModel').tag(sync=True)
+
+    morphTargets = Bool().tag(sync=True)
+    shading = Enum(['SmoothShading', 'FlatShading', 'NoShading'], 'SmoothShading').tag(sync=True)
+    wireframe = Bool().tag(sync=True)
```

Comment:
Well, maybe this is all right and we can just get used to the default being false.